bin/bower_ember_build

```diff
@@ -39,16 +39,17 @@ case $TRAVIS_BRANCH in
     CHANNEL="lts-2-4" ;;
   "release-1-13" )
     CHANNEL="release-1-13" ;;
-  "metal-views" )
-    CHANNEL="metal-views" ;;
-  "idempotent-rerender" )
-    CHANNEL="glimmer" ;;
   * )
     echo "Not a bower release branch.  Exiting!"
     exit 0 ;;
 esac
 echo -e "CHANNEL: ${CHANNEL}\n"
 
+if [[ -n $BUILD_TYPE ]]; then
+  CHANNEL=$BUILD_TYPE ;;
+  echo -e "CHANNEL OVERRIDE: ${CHANNEL}\n"
+fi
+
 # Sending output to /dev/null to prevent GH_TOKEN leak on error.
 git clone --branch ${CHANNEL} https://${USER}:${GH_TOKEN}@github.com/${COMPONENTS_EMBER_REPO_SLUG}.git bower_ember &> /dev/null
```

Comment:
Wouldn't this fail since glimmer is not a real branch? Maybe do the override after here?